cypress/support/providerCredentials.ts

```diff
@@ -0,0 +1,60 @@
+const providerCredentials = {
+  mxInt: {
+    username: Cypress.env('mx_client_id'),
+    password: Cypress.env('mx_api_secret'),
+    basePath: 'https://int-api.mx.com',
+    vcEndpoint: 'https://int-api.mx.com/',
```

Comment:
I know the code has a few places already where url variables are inconsistent whether they end with a "/" or not. In my opinion none should end in a slash. But it would be nice to establish a pattern and stick to it. What do you think?